### PR TITLE
[dom] Termgraph

### PR DESCRIPTION
**Pull request by jgphpc**

### CODE REVIEW

File: easybuild/easyconfigs/t/termgraph/termgraph-0.3.1-python3.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# jg (CSCS)
+easyblock = 'PythonPackage'
+
+name = 'termgraph'
+version = '0.3.1'
+_pymajver = '3'
+_pyshortver = '3.7'
```
[the remark this answers is not in the document]
Comment (jgphpc) [reply easybuild/easyconfigs/t/termgraph/termgraph-0.3.1-python3.eb:7]: Not with `easyblock = 'PythonPackage'` and cray-python/3.7.3.2
if you have a better solution, let me know

Comment (kraushm) [inline easybuild/easyconfigs/t/termgraph/termgraph-0.3.1-python3.eb:7]:
```suggestion
```

File: easybuild/easyconfigs/t/termgraph/termgraph-0.3.1-python3.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+versionsuffix = '-python%s' % _pymajver
```
Comment (kraushm) [inline easybuild/easyconfigs/t/termgraph/termgraph-0.3.1-python3.eb:8]:
```suggestion
versionsuffix = '-python%(pymajver)s'
```

File: easybuild/easyconfigs/t/termgraph/termgraph-0.3.1-python3.eb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+description = """A python command-line tool which draws basic graphs in the
+terminal"""
+toolchain = SYSTEM
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+dependencies = [('cray-python/3.7.3.2', EXTERNAL_MODULE)]
+unpack_sources = True
+download_dep_fail = True
+use_pip = True
+local_pythonpath = 'lib/python%s/site-packages' % _pyshortver
```
Comment (kraushm) [inline easybuild/easyconfigs/t/termgraph/termgraph-0.3.1-python3.eb:19]:
```suggestion
local_pythonpath = 'lib/python%(pyshortver)s/site-packages'
```

File: easybuild/easyconfigs/t/termgraph/termgraph-0.3.1-python3.eb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+exts_filter = ('python3 -c "import %(ext_name)s"', '')
+req_py_majver = _pymajver
```
Comment (kraushm) [inline easybuild/easyconfigs/t/termgraph/termgraph-0.3.1-python3.eb:21]:
```suggestion
req_py_majver = '%(pymajver)s'
```

File: easybuild/easyconfigs/t/termgraph/termgraph-0.3.1-python3.eb (hunk 2 above)
Comment (jgphpc) [reply easybuild/easyconfigs/t/termgraph/termgraph-0.3.1-python3.eb:8]: Can you poit me to your build dir please ? I get:
`== building and installing termgraph/0.3.1-python%(pymajver)s...`

**Comment by jgphpc** [comment]
thanks @kraushm

**Pull request merged**